app/addons/activetasks/tests/activetasks.componentsSpec.react.jsx

```diff
@@ -65,11 +65,21 @@ define([
       });
 
       afterEach(function () {
-        spy.restore();
+        if (spy) {
```

Comment:
I think what would be better is to have another afterEach in the __Active Tasks Filter tray__ tests and in there just do the `spy.restore` then you don't need this full if statement check.